examples/ads.amp.html

```diff
@@ -2448,6 +2449,16 @@ <h2>Viralize</h2>
       data-extra='{"lid":"22486","cid":"22572","preview":"1","f":"gallery"}'>
   </amp-ad>
 
+  <h2>vlyby</h2>
+  <amp-embed width="300" height="250"
```

Comment:
could we return an example ad here? It helps us debug, catch regressions etc

Reply:
yes, we will turn it back on, we just turned it off because there was strange amount of requests for test config